agent/.errcheck-exclude.txt

```diff
@@ -0,0 +1,2 @@
+(*database/sql.Rows).Close
```

Comment:
Why do we need these in the agent? It does not have SQL access.

Reply:
Like I said, I'm syncing the master and agent configs. Awkward enough having them duplicated without having them be slightly different as well.

Reply:
Given that they are two separate projects with separate needs, I think it is fine to have different configs to fit different needs. We are explicitly calling these as errors that we are going to ignore in the master. That does not necessarily mean these are errors that we would like to ignore in the agent as well.